### PR TITLE
Update router.ams1.yml

### PR DESCRIPTION
**Pull request by reulnl**
Added 4242421612

### CODE REVIEW

**Comment by yzguy** [comment]
@reulnl Thanks for submitting a peering, the validator identified that there is no valid A/AAAA record for the provided DNS record

File: routers/router.ams1.yml [hunk 1]
```diff
@@ -38,3 +38,14 @@
...
     public_key: 2VgphViJTNqQWlGII40Vfdjw+Uh6t/or1BHOQNDIS2A=
+
+- name: RAOULBROUNS-AMS01
+  asn: 4242421612
+  ipv6: fe80::5400:04ff:fecb:5ff1
+  multiprotocol: true
+  extended_nexthop: true
+  sessions: [ipv6]
+  wireguard:
+    remote_address: dn24bgp.raoulbrouns.nl
```
Comment (jvoss) [inline routers/router.ams1.yml:49]: Hi @reulnl - did you perhaps transpose `24` and `42`?

```suggestion
    remote_address: dn42bgp.raoulbrouns.nl
```

```
;; ANSWER SECTION:
dn42bgp.raoulbrouns.nl.	300	IN	AAAA	2a05:f480:1400:b4:5400:4ff:fecb:5ff1
```

**Review by jvoss — CHANGES_REQUESTED** [review]

File: routers/router.ams1.yml (hunk 1 above)
Comment (reulnl) [reply routers/router.ams1.yml]: Apologies, you're right. I've just committed the correct version.

Comment (jvoss) [reply routers/router.ams1.yml]: No problem, looks good to me now.

**Review by jvoss — APPROVED** [review]

**Comment by jvoss** [comment]
Thanks for the peering! Our information can be found under `router.ams1.routedbits.com` on our page: https://dn42.routedbits.com/nodes/

These changes will be pushed to our router shortly.

**Pull request merged**